am-lib/src/main/java/uk/gov/hmcts/reform/amlib/enums/Permissions.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.reform.amlib.enums;
+
+import java.util.List;
+
+public enum Permissions {
+    HIDE(0),
```

Comment:
shouldn't it start with 1 and following be a power of 2? (1, 2, 4, 8, 16 etc)

Reply:
I based on documentation on confluence for that, maybe Patryk will tell more about the approach.